### ISSUE TITLE
Private Access model (was: Index private messages yes/no)

### ISSUE THREAD

**Issue by arjenpdevries** [laura-dietz/soot#3]
The default now also indexes messages exchanged between people in private (enveloppe icon).
Maybe make that an option that needs to be turned on?

**Comment by laura-dietz** [comment]
Good idea. The whole access model is in flux. I am hoping for a contributor with experience that adopts this area.
